src/ukify/ukify.py

```diff
@@ -1055,19 +1056,19 @@ def parse_hwid_dir(path: Path) -> bytes:
     hwid_files = path.rglob('*.json')
 
     strings: set[str] = set()
-    devices: collections.defaultdict[tuple[str, str], set[uuid.UUID]] = collections.defaultdict(set)
+    devices: collections.defaultdict[tuple[int, str, str], set[uuid.UUID]] = collections.defaultdict(set)
```

Comment:
Because devtype in an integer, there aren't any checks on the format. If different type will have different lengths (which we kind of already want as DT has 2 fields and FW has 3) then ukify might create a broken UKI. 
Can you make devtype to be a string (`"devicetree"`, `"efifirmware"`, ...)?